NitroxPatcher/Main.cs

```diff
@@ -86,7 +86,7 @@ private static void InitPatches()
             Log.Info("Patching Subnautica...");
 
             // Enabling this creates a log file on your desktop (why there?), showing the emitted IL instructions.
-            HarmonyInstance.DEBUG = false;
+            HarmonyInstance.DEBUG = true;
```

Comment:
```suggestion
#if DEBUG
            HarmonyInstance.DEBUG = true;
#endif
```

Reply:
Like the comment said it will create a log file on the desktop so probably not a usually debug thing. Just if something is broken and then change this line manually.